### PR TITLE
chore: fix badge

### PR DESCRIPTION
**Pull request by fisker**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,12 @@
 # Sort Package.json
 
-[![Build Status](https://travis-ci.org/keithamus/sort-package-json.svg)](https://travis-ci.org/keithamus/sort-package-json)
+[![Build Status][github_actions_badge]][github_actions_link]
+[![NPM Version][package_version_badge]][package_link]
+
+[package_version_badge]: https://img.shields.io/npm/v/sort-package-json.svg
+[package_link]: https://www.npmjs.com/package/sort-package-json
+[github_actions_badge]: https://img.shields.io/github/actions/workflow/status/keithamus/sort-package-json/pr.yml
+[github_actions_link]: https://github.com/keithamus/sort-package-json/actions?query=workflow%3ACI+branch%3Amaster
```
Comment (fisker) [inline README.md:9]: @keithamus Since I'm working on this, do you think it's a good opportunity to rename `master` to `main`? Unless you don't want to.

Comment (keithamus) [reply README.md:9]: Yes please.

Comment (fisker) [reply README.md]: Done, https://github.com/keithamus/sort-package-json/pull/288/commits/4270583013242d13081665b6a65d10e08fe563f7.

Also made some changes to CI config.

I don't have access to rename branch, you'll have to do it.

Comment (keithamus) [reply README.md]: I've renamed to `main` :+1:

Comment (fisker) [reply README.md]: Good to go.

**Review by keithamus — APPROVED** [review]

**Comment by fisker** [comment]
Breaks the release script... https://github.com/keithamus/sort-package-json/actions/runs/4064399294/jobs/6997787276

**Pull request merged**